src/components/UsersList/index.jsx

```diff
@@ -48,97 +92,53 @@ export default class UsersList extends Component {
         })
       });
   }
-
-  nextItem = () => {
-    const {selectedPage, pageCount} = this.state;
-    if (selectedPage < pageCount){
-      Api.get(`/users?page=${selectedPage + 1}`)
-        .then(res => {
-          const {users, pageCount} = res.data;
-          this.setState({
-            users,
-            selectedPage: selectedPage + 1,
-            pageCount
-          })
-        });
-    }
-  }
-
-  previousItem = () => {
-    const {selectedPage} = this.state;
-    if (selectedPage > 1){
-      Api.get(`/users?page=${selectedPage - 1}`)
-        .then(res => {
-          const {users, pageCount} = res.data;
-          this.setState({
-            users,
-            selectedPage: selectedPage - 1,
-            pageCount
-          })
-        });
-    }
-  }
-
-  getPage = event => {
-    const page = parseInt(event.target.dataset.page);
-    Api.get(`/users?page=${page}`)
-      .then(res => {
-        const {users, pageCount} = res.data;
-        this.setState({
-          users,
-          selectedPage: page,
-          pageCount
-        })
-      });
+  
+  formatUsersForTable = (users) => {
+    const data = []
+    users.forEach((user) => {
```

Comment:
You should use .map instead of .forEach if you will create a new array

Reply:
Working on it.